src/eduid_webapp/email/views.py

```diff
@@ -184,9 +182,22 @@ def verify(user, code, email):
             '_status': 'error',
             'message': 'emails.code_expired_send_new'
         }
-
+    if code != state.verification.verification_code:
+        msg = "Invalid verification code for: {}".format(state.verification.email)
+        current_app.logger.debug(msg)
```

Comment:
Could skip the intermediate variable 'msg' here too.

Reply:
Done